app/main.py

```diff
@@ -1,5 +1,64 @@
+from __future__ import annotations
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot():
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        if coords is None:
+            self.coords = [0, 0]
+        else:
+            self.coords = coords
```

Comment:
It can be transformed into ternary operator:
```suggestion
        self.coords = [0, 0] if coords is None else coords
```
OR:
```suggestion
        self.coords = coords if coords else [0, 0]
```

Fix it in other cases as well, please.